tardis/adapters/sites/slurm.py

```diff
@@ -58,8 +58,14 @@ async def squeue(
         raise
 
     else:
+        # Not all lines returned in slurm_status.stdout belong to the actual squeue output
+        # Only keep lines containing the delimiter
+        cleaned_stdout = []
+        for row in slurm_status.stdout.splitlines():
+            if "|" in row:
+                cleaned_stdout.append(row)
```

Comment:
Use a comprehension instead of `for`-`append`.

```suggestion
        cleaned_stdout = [
            row for row in slurm_status.stdout.splitlines() if "|" in row
        ]
```